reana_resources_k8s/templates/config.yaml

```diff
@@ -0,0 +1,126 @@
+---
+# Deployment [local|prod]
```

Comment:
Maybe we should do it in a separate PR addressing https://github.com/reanahub/reana/issues/4 so we have the functionality shipped faster, WDYT?